config/utils/killProcess.js

```diff
@@ -1,14 +1,14 @@
 const ALLOWED_SIGNALS = ['SIGTERM', 'SIGINT', 'SIGQUIT'];
 
-export default (process, { signal = 'SIGTERM', timeout = 4000 } = {}) => {
+export default (process, { signal = 'SIGTERM', timeout = 15000 } = {}) => {
     if (!ALLOWED_SIGNALS.includes(signal)) {
         throw new Error(`Invalid signal: "${signal}", expecting one of ${ALLOWED_SIGNALS.join()}`);
     }
 
     return new Promise((resolve) => {
         const killTimeout = setTimeout(() => {
             // eslint-disable-next-line no-console
-            console.warn('[Vitamin] Server timeout: sending SIGKILL...');
+            console.warn(`[Vitamin] Process ${process.pid} timeout: sending SIGKILL...`);
```

Comment:
I am not really in favor of displaying a PID here. It might not be very user-friendly.